### PR TITLE
More polish about 3441 LWM2M Test Object

### PR DESCRIPTION
**Pull request by sbernard31**
This PR aims to improve/fix object 3441 triggered by #636.
It follow PR #639 and #642.

The name change is because of : https://github.com/OpenMobileAlliance/lwm2m-registry/pull/639#issuecomment-929388026.

@jpradocueva regarding https://github.com/OpenMobileAlliance/lwm2m-registry/issues/623#issuecomment-947245102, I get this is not advised to have several PR to add 1 object, so I'm not sure what would be the best way to contribute this work now my previous PR was already merged.

### CODE REVIEW

**Comment by jpradocueva** [comment]
@sbernard31 @mkgillmore the mismatch about `name` and `description`  between the DDF.xml file and the object 3441.xml has been resolved. The PR now passed validation.

File: 3441.xml [hunk 1]
```diff
@@ -30,8 +30,8 @@ POSSIBILITY OF SUCH DAMAGE.
 <LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
     <Object ObjectType="MODefinition">
-        <Name>LwM2M Specification Test Object</Name>
-        <Description1><![CDATA[This object contains resources for each datatype allowing easy interoperability testing.]]></Description1>
+        <Name>LwM2M v1.0 Test Object</Name>
+        <Description1><![CDATA[This object aims to make easier to do interoperability tests about LWM2M v1.0. It contains resources for each available datatype.]]></Description1>
```
Comment (hannestschofenig) [inline 3441.xml]:
```suggestion
        <Description1><![CDATA[This object is for use in interoperability tests of the LwM2M v1.0 technical specification. It contains resources for each available datatype.]]></Description1>
```

File: 3441.xml [hunk 2]
```diff
@@ -101,15 +101,15 @@ E.g. If you send an Exec /3441/0/3 with "3='stringValue',4" as arguments value,
...
+            <Item ID="4">
                 <Name>Arguments List</Name>
                 <Operations>R</Operations>
                 <MultipleInstances>Multiple</MultipleInstances>
                 <Mandatory>Optional</Mandatory>
                 <Type>String</Type>
                 <RangeEnumeration />
                 <Units />
-                <Description> List of Arguments from last execute on "Exec With Arguments"(3) resource.
+                <Description>List of Arguments from last execute on "Exec With Arguments"(3) resource. This resource is not affected by "Randomize values"(1) executable resource.
```
Comment (hannestschofenig) [inline 3441.xml]:
```suggestion
                <Description>List of Arguments from last execute on "Exec With Arguments"(3) resource. This resource is not affected by the "Randomize values"(1) executable resource.
```

**Comment by jpradocueva** [comment]
@sbernard31 sorry, I missed your previous comment about the best way to contribute as the changes were only partial. Your previous PR was merged into the "dev" branch. Therefore, your PR was applied to the correct baseline. The problem arises when the baseline receives different PRs on the same Object. If the DDF.xml in the baseline was not prepared in advance the validation tool may not find all the information to correctly validate the Object.
Now the changes were correctly validated, you can see the commits that I did and this will give you an idea of what problems were identified. 
Today, Dec 7th the group will meet and hopefully will approve the PR.

**Comment by hannestschofenig** [comment]
This PR has been approved by the group.

**Comment by sbernard31** [comment]
Great ! Thx for reviewing it :pray:.
I will backport last fixes in #648

**Comment by sbernard31** [comment]
Uups too late #648 is already merged :-/

**Pull request merged**